prime-dotnet-webapi/Models/Plr/PlrExpertise.cs

```diff
@@ -0,0 +1,16 @@
+
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Prime.Models.Plr
+{
+    [Table("PlrExpertiseLookup")]
+    public class PlrExpertise : ILookup<string>
+    {
+        [Key]
+        public string Code { get; set; }
```

Comment:
We usually have a autoincrementing integer id for our lookup tables. Not sure if it is necessary in this case, but maybe would cause some issues doing it this way if we want to link to this table in the future?

Reply:
Followed same pattern of other *Lookup tables like LicenseLookup, Country, Province etc. they do not have incrementing Id either.